### PR TITLE
Adding stubs required for latest nginx

### PR DESCRIPTION
**Pull request by lealem47**
# Description

Adding stubs required for latest nginx (1.25.5)

# Testing

nginx unit tests

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

File: src/conf.c [hunk 1]
```diff
@@ -1599,4 +1599,33 @@ int wolfSSL_CONF_cmd_value_type(WOLFSSL_CONF_CTX *cctx, const char *cmd)
  * END OF CONF API
  ******************************************************************************/
 
+#if defined(WOLFSSL_NGINX)
```
Comment (julek-wolfssl) [inline src/conf.c:1602]: I don't like osp specific gating. Put it in ossl all or leave it in extra with no gating.

**Review by julek-wolfssl — CHANGES_REQUESTED** [review]

**Review by julek-wolfssl — APPROVED** [review]

**Comment by julek-wolfssl** [comment]
You also need to add the version to `nginx.yml` once the patch get merged to wolfssl-nginx.

**Comment by lealem47** [comment]
> You also need to add the version to `nginx.yml` once the patch get merged to wolfssl-nginx.

There is no new patch, the patch for 1.25.0 was sufficient to get things building & tests passing

**Comment by lealem47** [comment]
Retest this please

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**